src/scala/com/github/johnynek/bazel_deps/CoursierResolver.scala

```diff
@@ -0,0 +1,181 @@
+package com.github.johnynek.bazel_deps
+
+import coursier.{Fetch, Cache, Resolution}
+import coursier.util.Task
+import cats.{Monad, MonadError, Traverse}
+import cats.data.{Nested, NonEmptyList, Validated, ValidatedNel}
+import cats.implicits._
+import org.slf4j.LoggerFactory
+import scala.collection.immutable.SortedMap
+import scala.util.{Failure, Try}
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration.Duration
+
+class CoursierResolver(servers: List[MavenServer], ec: ExecutionContext, runTimeout: Duration) extends Resolver[Task] {
+  // TODO: add support for a local file cache other than ivy
+  private[this] val repos = Cache.ivy2Local :: servers.map { ms => coursier.MavenRepository(ms.url) }
```

Comment:
so will this share/use a local ivy cache? this would mean sbt-publish local i think might work for hacking on something thats a jar dependency. Which would be cleaner than other options.

Reply:
yeah, it should. That is a nice side effect...